blog/scrum-tickets/task.md

```diff
@@ -0,0 +1,13 @@
+# Task
+
+> Cooking an elephant demistified 🐘
+
+So, here you stay in the middle of a ~jungle~ meeting room. Your team has just received a bunch of user stories. The managers stare at you, wondering why are you still here. We've just told you what a user wants, so go and do it. But you look at your team and feel that there's about 5 implementation ideas flying around... 
+
+But first, how do you cook an elephant? One task at the time!
+
+We need something more then a user story to start coding. Let's say a story is about a user being able to create an order. For that we'll need to create a UI form and an API endpoint. We have a backend and a frontend developer ready to step in. But both of them need an API contract. That's one of the things that should be defined in a task. A task should describe an input and an output of the applied change from a technical perspective. But who is responsible for that?
+
+The person who writes the task must be competent enough to define contracts of an API, communication logic of a UI component and any other technical detail. So that's clearly somebody from an engineering team. I would name a software architect, an engineering team lead and a senior developer as good candidates. Task is a crucial ticket type and people in many roles depend on it. They should all participate in the task review. A QA should check if every case is covered and clearly described. A dev should check if the task is clear and complete. A BA should check if the task will fulfil the user desires.
```

Comment:
to define the contracts of an API, the communication logic of a UI component, and any other technical detail

Reply:
fulfill the user's desires